e2e/tests/PageModel.ts

```diff
@@ -200,7 +207,7 @@ export const PageModel = {
             continueModalButton: 'button[aria-label="Continue"]',
             saveReferenceModalButton: 'div[role=dialog] button[title="Save"]',
             deleteReferenceModalButton: 'div[role="dialog"] button[title="Delete"]',
-            deleteFirstReferenceButton: 'tr:first-of-type button[title="Delete"]',
+            deleteFirstReferenceButton: 'table[data-testid="references-table"] tr:first-of-type button[title="Delete"]',
```

Comment:
Is there a reason for doing this using a data attribute rather than finding the table itself. Need to consider accessibility and checking you can get to it without relying on data tags. I don't know enough so can't advise on this, think aria tags are better though.

Reply:
I've changed this from data-testid to aria-label and while doing that reworked the publication flow tests a bit.